pkg/crc/machine/hyperkit/driver.go

```diff
@@ -0,0 +1,47 @@
+package hyperkit
+
+import (
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/machine/config"
+	"github.com/code-ready/machine/libmachine/drivers"
+	"github.com/pborman/uuid"
+	"path/filepath"
+)
+
+type hyperkitDriver struct {
+	*drivers.BaseDriver
+	DiskImage     string
```

Comment:
Currently, the hyperkit driver is not using DiskImage, I think it's better if the exact location of the disk image is internal to the driver, so it can be removed.